src/commcare_cloud/ansible/roles/barman/templates/streaming-pg.conf.j2

```diff
@@ -0,0 +1,12 @@
+{% if hostname is defined %}
```

Comment:
shouldn't this be `if 'hostname' in hostvars[item]` or `if hostvars[item].get('hostname')`